.github/workflows/test.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - 'release/**'
+      - "release/**"
```

Comment:
?

Reply:
auto-correction 😅 not sure if it's needed so removed this change